### DISCUSSION TITLE
EzMotion 3in1 sensor: motion detection stay always as detected.

### DISCUSSION THREAD

**Discussion by Extreme79** [zwave-js/zwave-js#6565] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

What is happening?
After interview, sensor was detected properly, temperature was read normally, as light level too.
But motion stay always detected and never change, turning the motion sensor useless.

What did you expect to happen instead?
I expect the motion sensor detect motion and than turn back to the normal state.

  1. Include the motion sensor
  2. Try to use it
  3. No changes in motion

Anything else we should know?
I use zwaveJS in home assistant; I created an[ issue on home assistant](https://github.com/home-assistant/core/issues/105893) too, as istructons ask, because I don't know if the issue come from zwavejs core or at other level.
Checking the log I can see that the values are correctly sent, the value from 0 to 255 and then 255 to 0, that identify the motion and the stop motion.

### Software versions

**As copied from Home assitant integration diagnostic**
Versione del driver: 12.4.1
Versione del server: 1.34.0
URL del server: ws://core-zwave-js:3000

### Device information

Manufacturer: Express Controls
Model name: EZMotion Express Wireless 3-in-1
Node ID: 25
fw rev.: 3.6

(another one, same fw, node 23)

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [X] The log includes the problematic interaction with the device _(if applicable)_.

- [X] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwave_js (3).log](https://github.com/zwave-js/node-zwave-js/files/13741906/zwave_js.3.log)

**Comment by AlCalzone** [comment, marked as answer]
Node 25 is using Basic CC Set to communicate its changes. If it has the same firmware as node 23 in the other issue, it's supposed to have a Binary Sensor of type Motion, so HA expects that it uses that to communicate its state.

There is an [open PR](https://github.com/home-assistant/core/pull/105134) to add a Basic sensor, I'll add your example there for a device that needs this.

**Reply by Extreme79** [reply]
Yes each ezmotion has the same 3.6 firmware, they are identical devices.
So I only have to wait. 
I hope not too much because I use this sensor as virtual alarm system 😄 

Thank you for your time.

Edit: I just realized, re-reading, that there's a way to detect motion in the meanwhile.. the *hidden* basic sensor that appear in HA as a light switch! So I can check the "switch" status having the motion status.
Edit2: temporarily, in Home Assistant, I created a template helper, as a binary sensor, that appear as motion sensor, based on the basic "light" switch value.
